### DISCUSSION TITLE
Samples were not included.

### DISCUSSION THREAD

**Discussion by short-k1ng** [SMByC/AcATaMa#14] [Q&A]
Hello! I'm sorry I'm not really a coder but I don't know where else to turn. I'm using the plugin for a project and I need it for accuracy assessment. My raster has a NoData value of -1e+9, but I couldnt choose that option in the tabs prior so when it gets to the accuracy assessment portion, I think it turns my 0 value pixels into the NoData pixels and subsequently, this excludes those pixels from the assessment. What can I do to remedy this? I'm sorry if this isn't the place to ask.

**Comment by XavierCLL** [comment]
Hi @short-k1ng 

This is the correct place to ask questions. And yeah, you are right, right now the plugin only accepts integers as a nodata, I haven't thought of that scenario, this is definitely a bug and I'm going to fix it for the next release, thanks for reporting this issue!. Meanwhile, the solution for you is to convert the Nodata to a value that your data never has, for example the maximum number that Acatama accept as a nodata is  999999

**Reply by short-k1ng** [reply]
Hi! I'd like to update my query. So I converted my raster, the nodatas are 5s now. But somehow it's still not including samples that were created on 0 value pixels. It's treating the 0s as a nodata value even if I put the nodata value as 5, I think? What are your thoughts on this? The 788 samples on here were created on the 0 pixel value that I'm using.

![image](https://user-images.githubusercontent.com/110748578/183276307-a21236b7-87ed-4983-8250-2f66e33d35e9.png)

![image](https://user-images.githubusercontent.com/110748578/183276319-d11bec67-bded-4001-8f3d-ad49cb8250e2.png)

**Reply by XavierCLL** [reply]
mmm it is weird, it seems like the layer still has 0 as a nodata
* check the nodata value of the layer in Qgis: right click on the layer > properties > information > bands

for unset the nodata, I usually use gdal scripts for that like (in the terminal):
* gdal_edit.py layer.tif -unsetnodata

maybe you can use Qgis, reset the nodata to another value:
* raster > conversion > translate (assign a specific nodata value, like  10)

**Comment by XavierCLL** [comment]
The other way to ignore that nodata value is disabled it in the stratified random sampling table:

![image](https://user-images.githubusercontent.com/11801453/183270232-8f0917f4-88b3-4299-99d0-3d4ef3d5ac98.png)

**Reply by short-k1ng** [reply]
Thank you for the response! Unfortunately, it doesn't show up in the table so I think converting the NoData into something else is the way to go. 
![image](https://user-images.githubusercontent.com/110748578/183270434-264ff067-227b-4875-9ac1-701c80c0667d.png)
